src/main/java/com/ihren/processor/constant/Account.java

```diff
@@ -0,0 +1,25 @@
+package com.ihren.processor.constant;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import java.util.stream.Stream;
+
+@Getter
+@RequiredArgsConstructor
+public enum Account {
+    MAIN("1", "Main"),
+    COUPON("2", "Coupon"),
+    BASE("3", "Base"),
+    TOTAL("4", "Total");
+
+    private final String id;
+    private final String name;
+
+    public static String getNameById(String id) {
```

Comment:
Please don't hesitate to return optional. Enum should not decide what should happen if there is no enum